app/resources/views/pages/dominion/realm.blade.php

```diff
@@ -54,6 +54,10 @@
                                     <tr>
                                         <td class="text-center">{{ $i + 1 }}</td>
                                         <td>
+                                            @if ($dominion->isMonarch())
+                                                <i class="ra ra-queen-crown ra-lg text-red"></i>
```

Comment:
Do we want to color the crown based on RG (green) and EG (yellow)? Or just have it red like it was in DC? Why couldn't you tell guard status for monarchs back in DC anyway? Any ideas?

Reply:
I think you could still tell if they were in guard in DC. I kinda like it as separate icon :) you can also have two icons when you are a monarch in protection.